Pressing enter during config setup breaks the setup

<!-- Please search for open issues that relate to the same problem before opening a new one. -->

#### System information

- **OS & Version**: OSX
- **Go version:** : 1.16
- **razor-go version:** latest

#### Describe the bug
Skipping config setup (for eg., pressing enter on set buffer) breaks the setup.

#### Expected behaviour
There should be defaults that should be set if config is skipped


#### Backtrace

````
Enter gas multiplier value: 
1
Enter buffer percent: 

\n
Setting initial config...
Error: flag needs an argument: 'b' in -b
Usage:
  razor setconfig [flags]

Flags:
  -h, --help   help for setconfig

Global Flags:
  -b, --buffer int32            buffer percent
  -g, --gasmultiplier float32   gas multiplier value (default -1)
  -p, --provider string         provider name

flag needs an argument: 'b' in -b
make: *** [set_config] Error 1````

<!-- When submitting logs: please submit them as text and not screenshots. -->

--- a/Makefile
+++ b/Makefile
@@ -14,16 +14,11 @@ fetch_bindings:
 install_razor:
 	@echo "Installing razor node...."
 	${GO} build -o ./build/bin/razor main.go
-	@echo "Razor node installed. \n"
+	@echo "Razor node installed."
+	@echo ""
 
 set_config:
-	@echo "Enter provider: "; \
-	read PROVIDER; \
-	echo "Enter gas multiplier value: "; \
-	read GAS_MULTIPLIER; \
-	echo "Enter buffer percent: "; \
-    read BUFFER_PERCENT; \
-    echo "\n"; \
-    echo "Setting initial config..."; \
-    ${RAZOR} setconfig -p $${PROVIDER} -g $${GAS_MULTIPLIER} -b $${BUFFER_PERCENT}
-	@echo "Setup done"
+	@echo "Setup initial config"
+	@${SHELL} config.sh
+	@echo ""
+	@echo "Razor node is set up and ready to use"

--- a/cmd/setconfig.go
+++ b/cmd/setconfig.go
@@ -1,9 +1,9 @@
 package cmd
 
 import (
+	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
-	"log"
 	"razor/utils"
 )
 
@@ -25,6 +25,12 @@ Setting the gas multiplier value enables the CLI to multiply the gas with that v
 		}
 		if bufferPercent != 0 {
 			viper.Set("buffer", bufferPercent)
+		}
+		if provider == "" && gasMultiplier == -1 && bufferPercent == 0 {
+			viper.Set("provider", "http://127.0.0.1:8545")
+			viper.Set("gasmultiplier", 1.0)
+			viper.Set("buffer", 30)
+			log.Info("Config values set to default. Use setconfig to modify the values.")
 		}
 		path := utils.GetDefaultPath() + "/razor.yaml"
 		err := viper.WriteConfigAs(path)

--- a/config.sh
+++ b/config.sh
@@ -1,0 +1,26 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+BIN_DIR=./build/bin
+RAZOR=${BIN_DIR}/razor
+
+read -rp "Provider: (http://127.0.0.1:8545) " PROVIDER
+if [ -z "$PROVIDER" ];
+then
+  PROVIDER="http://127.0.0.1:8545"
+fi
+
+read -rp "Gas Multiplier: (1.0) " GASMULTIPLIER
+if [ -z "$GASMULTIPLIER" ];
+then
+  GASMULTIPLIER=1.0
+fi
+
+read -rp "Buffer Percent: (20) " BUFFER
+if [ -z "$BUFFER" ];
+then
+  BUFFER=20
+fi
+
+$RAZOR setconfig -p $PROVIDER -b $BUFFER -g $GASMULTIPLIER

